src/measure_theory/hausdorff_measure.lean

```diff
@@ -546,4 +550,32 @@ lemma measure_zero_of_dimH_lt {μ : measure X} {d : ℝ≥0}
   μ s = 0 :=
 h $ hausdorff_measure_of_dimH_lt hd
 
+@[mono] lemma dimH_mono {s t : set X} (h : s ⊆ t) : dimH s ≤ dimH t :=
+bsupr_le $ λ d hd, le_dimH_of_hausdorff_measure_eq_top $
+  top_unique $ hd ▸ measure_mono h
+
+@[simp] lemma dimH_Union [encodable ι] (s : ι → set X) :
+  dimH (⋃ i, s i) = ⨆ i, dimH (s i) :=
+begin
+  refine le_antisymm (bsupr_le $ λ d hd, _) (supr_le $ λ i, dimH_mono $ subset_Union _ _),
+  contrapose! hd,
+  have : ∀ i, μH[d] (s i) = 0,
+    from λ i, hausdorff_measure_of_dimH_lt ((le_supr (λ i, dimH (s i)) i).trans_lt hd),
+  rw measure_Union_null this,
+  exact ennreal.zero_ne_top
+end
+
+lemma dimH_bUnion {s : set ι} (hs : countable s) (t : ι → set X) :
```

Comment:
Why didn't you put a simp attribute on this one (and the next ones)?

Reply:
No good reason.
```suggestion
@[simp] lemma dimH_bUnion {s : set ι} (hs : countable s) (t : ι → set X) :
```